src/main/scala/io/circe/yaml/package.scala

```diff
@@ -0,0 +1,112 @@
+package io.circe
+
+import scala.collection.JavaConverters._
+
+import cats.implicits._
+import org.yaml.snakeyaml.nodes._
+
+package object yaml {
+
+  private[this] def scalarNode(tag: Tag, value: String) = new ScalarNode(tag, value, null, null, '\0')
+
+  private[circe] def yamlTag(json: Json) = json.fold(
+    Tag.NULL,
+    _ => Tag.BOOL,
+    number =>
+      number match {
+        case JsonDecimal(_) | JsonBigDecimal(_) | JsonBiggerDecimal(_) | JsonDouble(_) =>
```

Comment:
What do you think about using `number.toBiggerDecimal.isWhole` here? These constructors aren't really intended for use outside circe proper (they're only available here because you're in `io.circe`), and I've been thinking recently about making some changes to them on the assumption that they're not being used externally.